src/apps/chifra/internal/globals/options.go

```diff
@@ -27,6 +27,10 @@ type GlobalOptions struct {
 	Version bool   `json:"version,omitempty"`
 	Noop    bool   `json:"noop,omitempty"`
 	NoColor bool   `json:"noColor,omitempty"`
+	// This flag is used in combination with --file. It informs,
+	// that we created JsonWriter that should not be passed to
+	// the next command from the file.
+	MixedFormats bool `json:"mixedFormats,omitempty"`
```

Comment:
There's an issue in our docs repo saying that we explicitly refuse to support this (mixed formats). I wonder if the complication is (if there actually is any) is worth it. Would people actually do this (mix formats)?

Although, now that I think of it, perhaps someone would do something likes this:

```
chifra export <some options> --fmt json --output file.json
chifra export <some options> --fmt csv --output file.csv
```

In other words, produce data in two formats for the exact same options. So, I guess, keep it.